fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/logical/LogicalAggregationOperator.java

```diff
@@ -84,7 +84,7 @@ public LogicalAggregationOperator(
         this.groupingKeys = ImmutableList.copyOf(groupingKeys);
         this.partitionByColumns = partitionByColumns;
         this.aggregations = ImmutableMap.copyOf(aggregations);
-        this.isSplit = isSplit;
+        this.isSplit = !type.isGlobal() || isSplit;
```

Comment:
to promise only Global type use un-split, because other type always come from `SplitAggregateRule` now, you can think that it was not set correctly before